Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -481,6 +481,8 @@
 Run2022B={355130: [[1, 40]]}
 steps['RunSingleMuon2022B']={'INPUT':InputInfo(dataSet='/SingleMuon/Run2022B-v1/RAW',label='2022B',events=100000,location='STD', ls=Run2022B)}
 steps['RunZeroBias2022B']={'INPUT':InputInfo(dataSet='/ZeroBias/Run2022B-v1/RAW',label='2022B',events=100000,location='STD', ls=Run2022B)}
+Run2022B={355680: [[21, 24]]}
```

Comment:
This was not in  #39217: can you add a comment?

Reply:
well, it's needed because the PR that added them was not backported.